src/components/common/v2/modal/MainSettingModal.tsx

```diff
@@ -0,0 +1,98 @@
+import styled from '@emotion/styled';
+import { useState } from 'react';
+
+import Button from '@/components/common/v2/button/Button';
+import DropdownButton from '@/components/common/v2/control/DropdownButton';
+import IconButton from '@/components/common/v2/IconButton';
+import DeadlineBox from '@/components/common/v2/popup/DeadlineBox';
+import PopUp from '@/components/common/v2/TextBox/PopUp';
+import { StatusType } from '@/types/tasks/taskType';
+
+function MainSettingModal() {
+	const [status, setStatus] = useState<StatusType>('미완료');
+
+	const handleStatusChange = (newStatus: StatusType) => {
+		setStatus(newStatus);
+	};
+
+	return (
+		<MainSettingModalLayout>
+			<MainSettingModalHeadLayout>
+				<ModalTopButtonBox>
+					<DropdownButton status={status} handleStatusChange={handleStatusChange} />
+					<ButtonBox>
+						<IconButton iconName="IcnDelete" type="normal" size="small" disabled />
+						<IconButton iconName="IcnX" type="normal" size="small" disabled />
+					</ButtonBox>
+				</ModalTopButtonBox>
+				<PopUp type="title" />
+			</MainSettingModalHeadLayout>
+			<MainSettingModalBodyLayout>
+				<DeadlineBox date={new Date()} endTime="06:00pm" label="마감 기간" />
+				<PopUpTitleBox>
+					<PopUp type="description" />
+				</PopUpTitleBox>
+				<DeadlineBox date={new Date()} startTime="11:00am" endTime="06:00pm" label="진행 기간" />
```

Comment:
```suggestion
				<DeadlineBox date={new Date()} endTime="06:00 pm" label="마감 기간" />
				<PopUpTitleBox>
					<PopUp type="description" />
				</PopUpTitleBox>
				<DeadlineBox date={new Date()} startTime="11:00 am" endTime="06:00 pm" label="진행 기간" />
```

시간 수정할 때 정렬이 살짝 이상해서 봤더니 사이에 띄어쓰기가 업서요 수정해주세요!!

Reply:
여기 나중에 데이터 임시로 넣어둔 부분이라서 나중에 데이터 넣으면서 수정하도록 하겠습니다!

Reply:
위에 버튼 정렬에서 제가 보기에는 정렬이 맞게 보이는데 이상하네용 ,,ㅜ
<img width="567" alt="image" src="https://github.com/user-attachments/assets/7fde4e6a-59aa-4381-81d9-24946f31b2d7" />